TLC5955.cpp

```diff
@@ -193,29 +193,11 @@ void TLC5955::setControlModeBit(bool is_control_mode)
 
 void TLC5955::updateLeds()
 {
-  // Get number of counts for current pattern
-  uint32_t power_output_counts = 0;
-  for (int16_t chip = (int8_t)_tlc_count - 1; chip >= 0; chip--)
-    for (int8_t led_channel_index = (int8_t)LEDS_PER_CHIP - 1; led_channel_index >= 0; led_channel_index--)
-      for (int8_t color_channel_index = (int8_t)COLOR_CHANNEL_COUNT - 1; color_channel_index >= 0; color_channel_index--)
-        power_output_counts += _grayscale_data[chip][led_channel_index][color_channel_index];
-  if (power_output_counts == 0) {
```

Comment:
This special case was removed. I think that we should re-instate it. It does reduce the power.

I think we should continue to turn off the gsclk when output power is identically zero

![image](https://user-images.githubusercontent.com/90008/145042334-90ec344b-8d84-4def-9bc6-b1be73e6cfdf.png)

![image](https://user-images.githubusercontent.com/90008/145042286-63cf3e99-d841-4803-bcde-7dbe8dd690d0.png)

Reply:
addressed